docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,403 @@
+---
+title: Host Certificates with Machine ID
+description: Issuing Host Certificates with Machine ID
+---
+
+In this guide, you will learn how to create host certificates through Machine ID for OpenSSH servers. This allows for the benefits of
+Machine ID with OpenSSH nodes and reduces risk by ensuring that short-lived certificates issued to tbot users on Teleport clusters can be successfully applied.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+(!/docs/pages/includes/tctl.mdx!)
+
+- OpenSSH server `sshd` version 6.9 or above installed. The SSH port on this host must be open to traffic from the Teleport Proxy Service host.
+
+  ```code
+  $ ssh -V
+  ```
+
+- A host that you wish to assign an identity to using Machine ID.
```

Comment:
Is this really a separate prerequisite? The host you're assigning an identity to is the OpenSSH server mentioned above.

Reply:
Removing this

Reply:
Removing, this is a leftover step borrowed from our Machine ID getting started guide. It shouldn't be necessary in this context